src/questions/smart_contracts.yaml

```
@@ -0,0 +1,966 @@
+  answerKeys:
+    - C
+  subTopics:
+    - "ethereum-transactions-working"
+  difficultyLevel: "Low"
+  choices:
+    - content: Transaction steps occur one by one in a sequence.
+      key: A
+    - content: Transactions need to be of a small amount.
+      key: B
+    - content: Transactions are reversed if a clause fails in a smart contract.
+      key: C
+    - content: Transactions take very less time in the Ethereum network.
+      key: D
+- uuid: b5c5ce04-ae7d-4c57-9071-119565bf5bcb
+  type: MultipleChoice
+  content: What are the properties of  Ethereum Transactions?
+  hint: NoHint
+  explanation: Ethereum transactions occur on P2P (Peer-to-Peer) basis. These transactions are irreversible once executed and they are atomic in nature.
+  answerKeys:
+    - A
+    - B
+    - C 
+  subTopics:
+    - "ethereum-transactions-working"
+  difficultyLevel: "Low"
+  choices:
+    - content: Peer to Peer
+      key: A
+    - content: Irreversible
+      key: B
+    - content: Atomic
+      key: C
+    - content: Involves middlemen
+      key: D
+- uuid: 98216c77-af95-40ce-b52f-a9853d96be88
+  type: SingleChoice
+  content: An Externally owned account can be used to transfer......
+  hint: NoHint
+  explanation: An EOA can be used to send any tokens within the Ethereum Network.
+  answerKeys:
+    - D
+  subTopics:
+    - "ethereum-transactions-working"
+  difficultyLevel: "Low"
+  choices:
+    - content: Bitcoin
+      key: A
+    - content: Ethereum
+      key: B
+    - content: DogeCoin
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 137cc95f-6c71-4ad6-83d9-1ec774488135
+  type: SingleChoice
+  content: An Ethereum account consists of how many fields?
+  hint: NoHint
+  explanation: An Ethereum account composes of majorly 4 fields. Those are nonce, balance, codeHash and storageRoot.
+  answerKeys:
+    - B
+  subTopics:
+    - "ethereum-transactions-working"
+  difficultyLevel: "High"
+  choices:
+    - content: 5
+      key: A
+    - content: 4
+      key: B
+    - content: 3
+      key: C
+    - content: 6
+      key: D
+- uuid: 63dd8506-1b0a-4b7e-8151-9e5973833231
+  type: SingleChoice
+  content: How is the transaction data stored in the blockchain?
+  hint: Working of transactions in the blockchain
+  explanation: The transaction data is stored in the form of a block in the blockchain.
+  answerKeys:
+    - A
+  subTopics:
+    - "ethereum-transactions-working"
+  difficultyLevel: "Low"
+  choices:
+    - content: Blocks
+      key: A
+    - content: Smart contracts
+      key: B
+    - content: Cryptographic hashes
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 3730622f-41b5-43ba-9b13-fbc0fc601b43
+  type: SingleChoice
+  content: Ethereum accommodates how many types of transactions?
```

```suggestion
  content: Ethereum accommodates which of the below types of transactions?
```

We should give the options as the types

---

done